docs/graphqlserve/graphqlserve.md

```diff
@@ -0,0 +1,171 @@
+---
+id: graphqlserve
+title: Running an API without code
+sidebar_label: API without code
+---
+
+With GraphQL Serve you can have a full featured [GraphQL CRUD](https://graphqlcrud.org/) API with subscriptions and data synchronization running in just a few seconds without writing a single line of code - all you need is a data model `.graphql` file.
+
+GraphQL Serve is a CLI tool that leverages the power of Graphback to generate a codeless Node.js GraphQL API complete with schema and CRUD resolvers and an in-memory MongoDB database.
+
+## Installation
+
+You can install `graphql-serve` globally with npm:
+
+```shell
+npm i -g graphql-serve
```

Comment:
For cli tools that do not have updates people recommend using npx. this way you will ensure that people using latest versions ( and we will get reliable stats about usage ;) )

Reply:
Do you see a section on usage with npx?